apps/client/src/app/pages/simulator/components/macro-popup/macro-popup.component.ts

```diff
@@ -103,6 +109,30 @@ export class MacroPopupComponent implements OnInit {
     if (this.aactionsMacro.length > 0) {
       this.aactionsMacro.push('/echo Cross class setup finished <se.4>');
     }
+
+    const necessaryBuffs = []
+    if (this.food) {
+      let foodBuff = this.i18n.getName(this.l12n.getItem(this.food.itemId));
+      if (this.food.hq) {
+        foodBuff += " HQ"
```

Comment:
I went ahead and used the `SIMULATOR.Hq` translation that someone had suggested. I did not find a string that already had something like `{{item}} HQ` that would allow me to make the space itself language sensitive. My gut says that should probably be its own thing to fix that across the site, so going to leave that out for now